### DISCUSSION TITLE
New install warning says to remove RainLab.Builder when it's not been installed

### DISCUSSION THREAD

**Discussion by Tony-S** [wintercms/winter#241] [Q&A] [answered]
I've just done a brand new install of Winter using composer. I've added various plugins using e.g. php artisan plugin:install Winter.User
In the backend I get the following warning:
```
Some issues have been detected and require attention:
The Winter.Builder plugin replaces RainLab.Builder, please remove RainLab.Builder to ensure there are no conflicts
The Winter.User plugin replaces RainLab.User, please remove RainLab.User to ensure there are no conflicts
```
I have not installed the Rainlab versions. They don't appear in the list of plugins or in the manage plugins page and they don't show as files in the plugins files area. Is this warning therefor a bug? Things seem to be working otherwise, I can create users ok.
thanks

**Comment by Tony-S** [comment]
Hi,

As per post, I used e.g. php artisan plugin:install Winter.User

Thanks

**Comment by mjauvin** [comment, marked as answer]
This problem is resolved in develop branch of winter. Release 1.1.4 will contain this fix.

**Comment by Tony-S** [comment]
Thanks
